CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ and this project adheres to
 
 ## [Unreleased]
 
+### Fixed
+
+- 🐛(frontend) fix update accesses form
```

Comment:
Usually we add the PR number at the end, it will create a natural link in Github when we add it to the release.

Reply:
OK good to know !